examples/cluster/plot_face_segmentation.py

```diff
@@ -26,26 +26,30 @@
 
 import numpy as np
 import scipy as sp
+from scipy.ndimage.filters import gaussian_filter
```

Comment:
I would use scikit image:

http://scikit-image.org/docs/dev/api/skimage.filters.html#skimage.filters.gaussian

Reply:
I found that as well. The problem with it is that it changed both name and module at different times since version 0.9.3 (which is the default in Ubuntu trusty) and thus one would need multiple `try:... except ImportError:...` which did not seem right for a simple example to me. (See the 3rd additional comment in opening post of the PR).
Do you agree that I then should keep SciPys version or should I change it anyway?